tests/playbooks/tests_bond_options.yml

```diff
@@ -128,7 +128,11 @@
                   arp_ip_target: 192.0.2.128
                   arp_validate: none
                   primary: "{{ dhcp_interface1 }}"
-
+                # add an infiniband port to the bond
+              - name: ib0
```

Comment:
The comment is indented wrong. Thanks!

Reply:
> I think this would cause problems as we do not have infiniband NICs on our CI.. so I think this would not work as we expect..

yes, but for upstream testing, we can check that the infiniband profile is created/present.